bash/code_to_gast/bash_router.py

```diff
@@ -0,0 +1,12 @@
+import bashlex
+
+def node_to_gast(node):
+
+    print (type(node))
```

Comment:
Remove?

Reply:
I removed that in one of the later commits